v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -89,55 +141,65 @@ export const MovableLine = (props: {
         fixEndPoints(lineObject.lower, 1)
         fixEndPoints(lineObject.middle, 2)
         fixEndPoints(lineObject.upper, 3)
-
+        fixHandles(lineObject.handleLower, 1)
+        fixHandles(lineObject.handleMiddle, 2)
+        fixHandles(lineObject.handleUpper, 3)
         refreshEquation()
       })
       return () => disposer()
-    }, [pointsOnAxes, lineObject, transform, xScale, yScale, model, xAxis, yAxis, xRange, yRange]
+    }, [pointsOnAxes, lineKey, lineObject, plotHeight, plotWidth, transform, 
+        xScale, yScale, model, xAttrName, xSubAxesCount, xAxis, yAttrName, 
+        ySubAxesCount, yAxis, xRange, yRange]
```

Comment:
This long list of dependencies says that if any of these values change, the `autorun` will be disposed and reinstalled. It would be better to let the existing `autorun` respond to any changes instead, which can be done by dereferencing the relevant values inside the `autorun` rather than outside the `useEffect()`. Just something to think about moving forward.